src/components/exercise/index.cjsx

```diff
@@ -135,12 +136,15 @@ ExerciseWithScroll = React.createClass
 
   render: ->
     {parts, footer, pinned} = @props
+    classes = classnames('openstax-multipart-exercise-card', {
+      "deleted-homework": @props.task.type is 'homework' and @props.task.is_deleted
+    })
```

Comment:
could we do `@props.task?.type` and `@props.task?.is_deleted` in case `task` does not get passed in?  for example in concept coach